src/components/Nav.tsx

```diff
@@ -96,11 +97,28 @@ export const NavHeader = () => {
       px={3}
     >
       <Flex alignItems="center">
-        {/* TODO: Eventually this will be replaced with a logo or something nicer */}
-        <Text mr={4}>
-          <b>SchedulaterLogoHere</b>
-        </Text>
-        <NavLink to="/" desc="Home" />
+        {/*logo box*/}
+
+        <Box
+          mr="15px"        
+          bg="green.200"
+          h="40px"
+          w="40px"
+          borderRadius="50%"
+        >
+          <Image 
+            src="/logo.png" 
+            alt="schedulator logo" 
+            height="40px"
+            width="40px"
+            minWidth="40px"
```

Comment:
A bit nit picky but I don't think you actually need to specify height and width here, the logo should take the size of the box automatically. minWidth can be kept if you want to ensure the logo stays the same size when the window changes, however.

Reply:
Looks good otherwise!

Reply:
sweet thanks!